ds_judgements_public_ui/sass/includes/_service_introduction.scss

```
@@ -13,6 +13,20 @@
     margin: 0;
   }
 
+  &__header-beta {
+    font-size: 2.5rem;
+    line-height: 3.8rem;
+    display: inline-block;
+    width: 100%;
+    font-family: $font-roboto;
+    font-weight: normal;
+    color: $color-white;
+    margin-top: 2rem;
```

Looking at the designs, I think we'll need to add margin-bottom: 0; so it sits near the bottom.
```suggestion
    margin: 2rem 0 0;
```

---

@jlhdxw thanks added